build/gtkradiant/README.md

```diff
@@ -1,20 +1,12 @@
 Unvanquished support for GtkRadiant
 ===================================
 
-Supported versions
-------------------
-
-* GtkRadiant 1.6
-
 Installation
 ------------
 
 GtkRadiant automatically fetches gamepacks it supports, you have no need to clone this repo.
 
-List of available games is hardcoded, so you will need patched version.
-Use my repo https://github.com/neumond/GtkRadiant
```

Comment:
I have just uploaded my backup of this repo at https://github.com/Viech/GtkRadiant, just so you know.